### ISSUE TITLE
Missing depdendencies

### ISSUE THREAD

**Issue by codepainters** [codepainters/ise14#3]
Not everything works yet:
```
PlanAhead
  Exception in thread "main" java.lang.UnsatisfiedLinkError: /opt/Xilinx/14.7/ISE_DS/PlanAhead/tps/lnx64/jre/lib/amd64/xawt/libmawt.so: libXtst.so.6: cannot open shared object file: No such file or directory

FPGA Editor
  /opt/Xilinx/14.7/ISE_DS/ISE/bin/lin64/_fpga_editor: error while loading shared libraries: libXp.so.6: cannot open shared object file: No such file or directory

XPowerAnalyzer
  /opt/Xilinx/14.7/ISE_DS/ISE/bin/lin64/_xpa: error while loading shared libraries: libXcursor.so.1: cannot open shared object file: No such file or directory
```

Note: `libXp.so.6` is problematic, it's not available in Ubuntu 22.04 nor 20.04.

**Comment by codepainters** [comment]
It seems `libXp.so.6` is available up to and including Ubuntu 15.04 :-O

**Comment by codepainters** [comment]
I've switched to 14.04 LTS. Now I get:
```
/opt/Xilinx/14.7/ISE_DS/ISE/bin/lin64/_fpga_editor: error while loading shared libraries: libstdc++.so.5: cannot open shared object file: No such file or directory
```
